rawdata/misc/commanders_to_ach_generals.csv

```diff
@@ -0,0 +1,745 @@
+CS116,http://dbpedia.org/resource/Randall_L._Gibson,7171c4f3-6a55-4070-a7c9-86cf6fd8c183
+US49,http://dbpedia.org/resource/Edward_Canby,30a29258-21ca-4370-8924-ee4559fe526b
+US49,http://dbpedia.org/resource/Edward_Canby,30a29258-21ca-4370-8924-ee4559fe526b
+CS109,http://dbpedia.org/resource/Nathan_Bedford_Forrest,02b4fa9a-541a-45c7-9097-ab1aee538c6c
+US117,http://dbpedia.org/resource/James_H._Wilson,460c9e8f-625e-490f-8eda-7385907b8d3f
+CS27,http://dbpedia.org/resource/Earl_Van_Dorn,a68145ec-a562-494d-8e95-c9bd2e0ff473
+US176,http://dbpedia.org/resource/Samuel_Ryan_Curtis,e08b618c-acc7-4502-9ef8-b8546572e22d
+US200,http://dbpedia.org/resource/William_Plummer_Benton,291cca7a-a675-404f-9c4d-7c55d227e81e
+US34,http://dbpedia.org/resource/Charles_Edward_Hovey,c26f173d-b9e3-4afa-89d4-5928f663c844
+CS135,http://dbpedia.org/resource/Thomas_C._Hindman,839a5a85-49a8-41a6-b4a9-f76657c5d4fc
+CS151,http://dbpedia.org/resource/William_Henry_Parsons_(colonel),6370ad3b-9871-43c7-beb5-6db2824b3db7
+US108,http://dbpedia.org/resource/James_G._Blunt,a3f15046-9028-421d-bbe0-fb6c341a991f
+CS85,http://dbpedia.org/resource/John_S._Marmaduke,7b7bfdd4-920e-4d28-a2eb-44e33941d4b0
+CS135,http://dbpedia.org/resource/Thomas_C._Hindman,839a5a85-49a8-41a6-b4a9-f76657c5d4fc
+US68,http://dbpedia.org/resource/Francis_J._Herron,42af3182-e310-412a-917c-4718d4f4dda6
+US108,http://dbpedia.org/resource/James_G._Blunt,a3f15046-9028-421d-bbe0-fb6c341a991f
+US205,http://dbpedia.org/resource/William_Vandever,e40d06fb-a659-4f50-add2-8aebb8d9156b
+CS85,http://dbpedia.org/resource/John_S._Marmaduke,7b7bfdd4-920e-4d28-a2eb-44e33941d4b0
+US28,http://dbpedia.org/resource/Benjamin_Prentiss,84cac7af-39b5-4fda-af6a-303608eec2c3
+CS134,http://dbpedia.org/resource/Theophilus_H._Holmes,65f2acff-a023-45b9-8957-6b89af4343b7
+US197,http://dbpedia.org/resource/William_F._Cloud,6910b429-cf7d-4766-ae47-194be94c58b2
+CS143,http://dbpedia.org/resource/William_Lewis_Cabell,74670315-8624-4010-bd86-1e4e2d5596f3
+US142,http://dbpedia.org/resource/John_Davidson_(general),50a85216-8922-4d4f-93c9-e4fd108eb188
+CS85,http://dbpedia.org/resource/John_S._Marmaduke,7b7bfdd4-920e-4d28-a2eb-44e33941d4b0
+CS85,http://dbpedia.org/resource/John_S._Marmaduke,7b7bfdd4-920e-4d28-a2eb-44e33941d4b0
+US165,http://dbpedia.org/resource/Powell_Clayton,65299168-a5f5-4a3b-8daa-7a9c0468c210
+CS85,http://dbpedia.org/resource/John_S._Marmaduke,7b7bfdd4-920e-4d28-a2eb-44e33941d4b0
+US71,http://dbpedia.org/resource/Frederick_Steele,d84237f6-ba30-44d3-9aa9-7e2aafed4d9a
+US71,http://dbpedia.org/resource/Frederick_Steele,d84237f6-ba30-44d3-9aa9-7e2aafed4d9a
+CS133,http://dbpedia.org/resource/Sterling_Price,1cd425bc-311a-42c3-9ba2-a16bc1db7eea
+CS85,http://dbpedia.org/resource/John_S._Marmaduke,7b7bfdd4-920e-4d28-a2eb-44e33941d4b0
+CS128,http://dbpedia.org/resource/Samuel_B._Maxey,3f0622ad-44c0-47f0-ae02-25b81a5bad3a
+US111,NA,cb2cf7d8-0211-4695-99a1-4e3b3515b7ca
```

Comment:
@weszuidema Can you add the names of the individuals? It's not needed for merging, but its good for sanity and debugging.